Feature/12 hugo conversion (#14)
Cov-ComSec/Cov-ComSec.github.io: content/posts/welcome.md

---
title: "Welcome!"
date: 2021-05-25T23:14:41+01:00
lastmod: 2021-05-25T23:14:41+01:00
draft: false
author: "Jack Orcherton"
authorLink: "https://jackorcherton.github.io"
description: "A brief introduction to the new site and future plans"
tags: ["welcome "]
---

Welcome to the new website for ComSec! We hope you find it to be an improvement to the last, as we feel the new layout will make our content easier to reach. 

## Main Improvements
The main improvements to the site are:

> - dark theme- post tagging 

Reviewer: ```suggestion
- dark theme
- post tagging 
```

- clear author

> We also wish to encourage more users to post content (such as walk-throughs) and will recieve full accreditation. I will include more information on how to contribute in the near future. 

Reviewer: ```suggestion
We also wish to encourage more users to post content (such as walk-throughs) and they will receive full credit. I will include more information on how to contribute in the near future.
```

> So please bear with us while we migrate content and continue to update the website and feel free to point out any issues. Thanks, Jack and Comsec Committee Get Outlook for Android

Reviewer: ```suggestion
So please bear with us while we migrate content and continue to update the website, and feel free to point out any issues.

Thanks,

Jack and the rest of the Comsec Committee
```
